extensions/PhabBugz/lib/WebService.pm

```diff
@@ -61,7 +61,7 @@ sub revision {
 
     # If bug is public then remove privacy policy
     my $result;
-    if (!@{ $bug->groups_in }) {
+    if (is_public($bug)) {
```

Comment:
You will need to add to the top:

use Bugzilla::Extension::Push::Util qw(is_public);